packages/now-client/src/deploy.ts

```diff
@@ -19,13 +20,16 @@ export interface Options {
   isDirectory?: boolean;
   defaultName?: string;
   preflight?: boolean;
+  debug?: boolean;
 }
 
 async function* createDeployment(
   metadata: DeploymentOptions,
   files: Map<string, DeploymentFile>,
   options: Options
 ): AsyncIterableIterator<{ type: string; payload: any }> {
+  // const debug = createDebug(options.debug)
+  // delete options.debug
```

Comment:
Since it's commented we can probably remove it?

Reply:
Oh, actually we need to uncomment it 😅
Thanks for the catch!

Reply:
Or rather alter this a bit